qa/workunits/rados/test_alloc_hint.sh

```diff
@@ -51,6 +51,7 @@ function setup_pgid() {
 }
 
 function expect_alloc_hint_eq() {
+    export CEPH_ARGS="--osd-objectstore=filestore"
```

Comment:
this does not change how osd is initialized. you will need to update the`overrides` section `qa/suites/rados/objectstore/backends/alloc-hint.yaml` to specify the objectstore type.

and you need to audit other places you set `--osd-objectstore=filestore` to make sure it can actually take effect before the OSD/cluster is created.

Reply:
```
2018-11-29T02:58:03.956 INFO:tasks.workunit.client.0.smithi083.stderr:+ sudo ceph daemon osd.0 flush_journal
2018-11-29T02:58:04.143 INFO:tasks.workunit.client.0.smithi083.stderr:++ sudo sh -c 'ls /var/lib/ceph/osd/ceph-0/current/2.6*_head/foo_*'
2018-11-29T02:58:04.158 INFO:tasks.workunit.client.0.smithi083.stderr:ls: cannot access /var/lib/ceph/osd/ceph-0/current/2.6*_head/foo_*: No such file or directory
```
see http://pulpito.ceph.com/kchai-2018-11-29_01:22:06-rados-wip-kefu2-testing-2018-11-29-0053-distro-basic-smithi/3289928/